File: Ruby/vowels.rb
sentence = "The quick brown fox jumps over the lazy dog"
VOWELS = "AEIOUYaeiouy"

# Convert sentence string to array
sentence_array = sentence.split("")

# Initialize vowel counter
counter = 0

# Loop through sentence_array
for i in 1..sentence_array.length()-1 do

    # Check to see if VOWELS include the current character, and if so increase counter by 1
    if VOWELS.include?(sentence_array[i])
        counter += 1
    end
end

puts counter

#puts sentence_array

File: Dart/vowels.dart
main() {
    var sentence = "The quick brown fox jumps over the lazy dog";
    var VOWELS = "AEIOUYaeiouy";

    // Convert sentence string to a list (array)
    var sentence_array = sentence.split("");

    // Initialize vowels counter
    var counter = 0;

    // Loop through sentence_array
    for (var i = 0; i < sentence_array.length; i++){
        // If the current character is a vowel, increase the counter
        if(VOWELS.contains(sentence_array[i])){
            counter ++;
        }
    }

    // Print result
    print(counter);
}